proto/umee/leverage/v1/leverage.proto

```diff
@@ -12,22 +12,38 @@ message Params {
   option (gogoproto.goproto_stringer) = false;
   // Complete Liquidation Threshold determines how far over their borrow
   // limit a borrower must be in order for their positions to be liquidated
-  // fully in a single event.
+  // fully in a single event. See minimum_close_factor for more details.
+  // Valid values: 0-1.
   string complete_liquidation_threshold = 2 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable)   = false,
     (gogoproto.moretags)   = "yaml:\"complete_liquidation_threshold\""
   ];
-  // Minimum Close Factor determines the portion of a borrower's position
-  // that can be liquidated in a single event, when the borrower is just barely
-  // over their borrow limit.
+  // Close Factor determines the portion of a borrower's position that can be
+  // liquidated in a single event. Minimum Close Factor is Close Factor at
+  // liquidation_threshold. 0.1 means that that 10% of the borrower position can
+  // be liquidated when the borrowed value passes the/ liquidation_threshold.
+  // close_factor scales linearly between minimum_close_factor and 1.0,
+  // reaching its maximum when borrowed value passed
+  // complete_liquidation_threshold. We can put it into the picture:
+  //
+  //
+  //             borrowed           C := collateral
+  //             value                   value
+  //  --- | ------- | ----- | -------- | ------->
+  //   liquidation      complete
+  //   threshold        liquidation
+  //    * C             threshold * C
+  //
+  // Valid values: 0-1.
```

Comment:
```suggestion
  // reaching its maximum when borrowed value exceeds the limit defined by complete_liquidation_threshold.
  // Valid values: 0-1.
```

Reply:
I don't find them in other docs. I was proposing to add it in the liquidate.go.

Proto is the main entry point for inspecting parameters and queries. These should be well documented.

Reply:
In that case - we could keep the diagram but change the labels slightly:

If the `* C` is removed from liquidation threshold and complete liquidation threshold, and the `C := ` is removed from collateral value, it will no longer imply slightly misleading math.

Reply:
`complete liquidation threshold * C` does not equal the borrowed value where `close_factor = `.

For example, if
```
Liquidation Threshold = $60
Collateral value = $100
Complete_Liquidation_Threshold = 0.1
```

then the close factor becomes 1.0 when borrowed value = $64
```
$60 + ($100 - $60) * 0.1
```

Labeling the value as `Complete_Liquidation_Threshold * C` is misleading because `$64 != 0.1 * $100`

Reply:
OK, I made it more clear. Could you check again?